lib/src/_utils.js

```diff
@@ -191,9 +192,13 @@ function requireConfig(filepath) {
  * @param  {string} path Path where to look for config files
  * @return {Object} The configuration from the first found file or empty object
  */
-function getConfigFromFile(path) {
-    return getFileTypes()
-        .reduce((carry, filename) => carry || requireConfig(path + '/' + filename), false) || {};
+function getConfigFromFile(path, customFilename = null) {
+    if (customFilename) {
+        return requireConfig(Path.join(path, customFilename)) || {};
+    } else {
+        return getFileTypes()
```

Comment:
What do you think about about:

```javascript
return [].concat(customFilename, getFileTypes())
    .reduce() || {};
```

So that, if the user passes a wrong configuration file, gren will still look for existing ones.

Reply:
Good call! Since the user is passing a specific config filename in, they're expecting the file to exist — do you think it'd be more user-friendly to throw an error here and notify them that the file wasn't found?